src/io/flutter/utils/AndroidUtils.java

```diff
@@ -500,7 +500,7 @@ private void enable() {
         addIncludeStatement();
         if (errorDuringOperation.get()) return;
         addCoeditTransformedProject(project);
-        projectRoot.refresh(true, true);
+        projectRoot.refresh(false, true);
```

Comment:
note: it looks like this change forces the refresh to happen synchronously; perhaps add a comment in-line, so people will know that the use of that param is important?

Reply:
Done!